docs/source/release_notes/release-notes.rst

```diff
@@ -44,6 +44,15 @@ develop
     *    - Type
          - Change
 
+    *    - |devbreak|
```

Comment:
If I understand the PR correctly, this is still possible though, so it's not really a dev break.

Reply:
true.